stock_picking_yamato_csv/report/report_stock_picking_yamato_csv.py

```diff
@@ -263,6 +263,11 @@ def generate_csv_report(self, writer, data, pickings):
                 or partner_shipping.yamato_carrier_code
                 or warehouse.yamato_carrier_code
             )
+            if scheduled_date < today_date:
+                raise UserError(
+                    _("There is a delivery with a past scheduled date:\n%s")
+                    % ("\n".join(picking.mapped("name")))
```

Comment:
```suggestion
                    _("There is a delivery with a past scheduled date: %s")
                    % (picking.name)
```
There is no point joining the strings if we are looking at only one picking here.

Reply:
@yostashiro Thank you,  applied your change!